csrc/mmdeploy/net/torchscript/CMakeLists.txt

```diff
@@ -2,23 +2,27 @@
 
 project(mmdeploy_torch_net)
 
-find_package(Torch REQUIRED)
-find_package(TorchVision)
+option(MMDEPLOY_TORCHSCRIPT_SDK_BACKEND "Build TorchScript SDK backend" OFF)
```

Comment:
Can `MMDEPLOY_BUILD_SDK=ON` and `MMDEPLOY_TARGET_BACKENDS="torchscript"` be enough?

Reply:
If you just want the converter to work with torchscript (for optimization passes), this would introduce the chaos of different C++ ABIs.

Reply:
If only converter is concerned, we can disable `MMDEPLOY_BUILD_SDK`

Reply:
What if I need to use converter with optimizer to convert TensorRT models and use SDK with TensorRT backend?